scripts/variantstore/wdl/GvsImportGenomes.wdl

```diff
@@ -45,6 +50,10 @@ workflow GvsImportGenomes {
     File? load_data_gatk_override
     String? billing_project_id
     Boolean is_wgs = true
+
+    Boolean is_custom_ref = false
```

Comment:
doesn't seem to be used

Reply:
good point.  I put it there to choose between normal and custom behavior before remembering that I wasn't going to care about merging normal and custom behavior as part of this PR!